src/main/java/com/ksm/wstbackoffice/entity/Address.java

```diff
@@ -1,17 +1,38 @@
 package com.ksm.wstbackoffice.entity;
 
 import lombok.Data;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
 
+@Entity
 @Data
 public class Address {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
1. Which strategy do you know and what is the difference between them?
2. Why did you choose IDENTITY?

Reply:
Which strategy do you know and what is the difference between them?
answer:
1.Auto Strategy - Strategy uses the global number generator to generate a primary key for every new entity object.
These generated values are unique at the database

2.Identity Strategy - The IDENTITY strategy is very similar to the AUTO strategy also generates an automatic value during commit for every new entity object. The difference is that a separate identity generator is managed per type hierarchy, so generated values are unique only per type hierarchy.

3.Sequence Strategy - Unlike AUTO and IDENTITY, the SEQUENCE strategy generates an automatic value as soon as a new entity object is persisted

4.Table Strategy - The TABLE strategy is very similar to the SEQUENCE strategy.
A tiny difference is related to the initial value attribute. Whereas the SEQUENCE strategy maintains the next sequence number to be used the TABLE strategy maintains the last value that was used.

Why did you choose IDENTITY?
I copied from my previos project

I changed strategy from IDENTITY to AUTO I think its better

Reply:
I don´t want you to copy the text from google. I want you to understand different types.
Why did you choose AUTO?

Reply:
I dug deeper and changed my opinion again
I think that @GeneratedValue(strategy = GenerationType.IDENTITY) better then GenerationType.AUTO here is explenation -> 

https://ngdeveloper.com/generationtype-identity-vs-generationtype-sequence-vs-generationtype-auto/#:~:text=Things%20to%20Note%3A-,GenerationType.,starts%20its%20value%20from%200.

Reply:
I chose GenerationType.IDENTITY because 
1. GenerationType.Identity does not create any additional sequences / sequence tables.
2. Every table/entity created with GenerationType.IDENTITY starts its value from 0.

Reply:
unlike GenerationType.AUTO 
if use GenerationType.AUTO hibernate internally creates hibernate_sequence table to maintain the sequence number, the more important thing to consider is, it shares the sequence number across the tables.